### ISSUE TITLE
I'm getting lower results, Am I Evaluation correctly? (Kinetics)

### ISSUE THREAD

**Issue by BarakBat** [kenshohara/3D-ResNets-PyTorch#131]
I'm having the pretrained model from drive, and using this command:
"python main.py --model_depth 18 --dataset kinetics --model resnet --batch_size 16 --n_classes 400 --annotation _path kinetics_annotation.json --resume_path resnet-18-kinetics.pth --no_train"

I'm Evaluating by normal running just without training.

I'm getting around 39% In acc column which suppose to be 54%.
Happens also in Resnext101, and in Resnet34.

What am I doing wrong?!
Does anyone have any Idea?

Thanks!

**Comment by BarakBat** [comment]
> I'm having the pretrained model from drive, and using this command:
> "python main.py --model_depth 18 --dataset kinetics --model resnet --batch_size 16 --n_classes 400 --annotation _path kinetics_annotation.json --resume_path resnet-18-kinetics.pth --no_train"
> 
> I'm Evaluating by normal running just without training.
> 
> I'm getting around 39% In acc column which supposes to be 54%.
> Happens also in Resnext101, and in Resnet34.
> 
> What am I doing wrong?!
> Does anyone have any Idea?
> 
> BTW: Maybe It's because in validation set I'm getting clip accuracy and not video accuracy?
> 
> I tried another method to Evaluate, in kinetics I'm getting 0% and in HMDB51, I'm getting even Higher results , what I'm doing is:
> 
> 1. Testing model
> 
python main.py --model_depth 101 --dataset hmdb51 --video_path .jpg --model resnext --batch_size 16 --n_classes 51 --annotation_pathannotation_dir_path/hmdb51/hmdb51_1.json --pretrain_path resnext-101-kinetics-hmdb51_split1.pth --no_train --no_val --test

> 2.evaluating the model:
> ` python utils/eval_hmdb51.py`
> 
> I added a main function :
> 
> ```
> if __name__ == "__main__":
>     hmdb = KINETICSclassification('annotation_dir_path/hmdb51/hmdb51_1.json',
>                                   'val_hmdb51.json', verbose=True, top_k=1)
>     hmdb.evaluate()
> ```
> I'm getting 75% and not 63% (top1) on Hmdb51 (avarage on all splits: the error@1 is :(0.36,0.19,0.19), kinetics I'm getting Error@1 =1.1
> 
> Thanks!

**Comment by MenghaoGuo** [comment]
I have the same problem.

**Comment by HaiyiMei** [comment]
Hey, I got the same problem. Have you guys fixed it?
Which version of pytorch are you using? I'm running the code in pytorch 1.0.0 / CUDA 9.
